src/pages/Homepage.jsx

```diff
@@ -0,0 +1,12 @@
+import { Categories, Hero } from "components";
+import React from "react";
+import { Fragment } from "react/cjs/react.production.min";
```

Comment:
importing from minified scripts can lead to errors, make sure auto imports are importing from correct path

Reply:
use <></> instead of fragment